### DISCUSSION TITLE
addon Menu-o-Matic - new release

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/freeplane#368] [Show and tell]
hi,
a new release from [Menu-o-Matic](https://github.com/EdoFro/Freeplane_Menu-o-Matic) can be found [here](https://github.com/EdoFro/Freeplane_Menu-o-Matic/releases/tag/v0.1.3)

bye,

edo

**Comment by Illegit** [comment]
Hi EdoFro, is it possible to apply the same approach as for Tool Panel addon menu, to some of the menu-o-matic solutions?  In order to have the possibility to have some of them already open when you startP FP.
Thanks.

**Reply by EdoFro** [reply]
Hi Illegit,
I think this is possible, but we have to define the feature in more detail.
I created a [Feature Request in MoM repository](https://github.com/EdoFro/Freeplane_Menu-o-Matic/issues/2#issue-1834111483), where we can discuss how this feature should work. 
 ok?

**Reply by EdoFro** [reply]
Done. Look at [this](https://github.com/EdoFro/Freeplane_Menu-o-Matic/issues/2#issuecomment-1670101619)

**Comment by Illegit** [comment]
Nice.Thanks
